src/org/opendatakit/briefcase/util/TransferFromODK.java

```diff
@@ -21,7 +21,6 @@
 import java.io.IOException;
 import java.nio.file.Path;
 import java.sql.SQLException;
-import java.util.Optional;
```

Comment:
old/legacy

To be replaced by a new implementation along with `PullFromCentral` and `PullFromAggregate`